src/plugins/interactive_setup/public/enrollment_token_form.tsx

```diff
@@ -226,3 +224,59 @@ export function compareAddresses(a: string, b: string) {
   }
   return 0;
 }
+
+export const EnrollmentTokenHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const button = (
+    <EuiLink onClick={() => setIsPopoverOpen((isOpen) => !isOpen)}>
+      <FormattedMessage
+        id="interactiveSetup.enrollmentTokenHelpPopover.buttonText"
+        defaultMessage="Where do I find this?"
+      />
+    </EuiLink>
+  );
+
+  return (
+    <EuiPopover
+      button={button}
+      anchorPosition="rightCenter"
+      isOpen={isPopoverOpen}
+      closePopover={() => setIsPopoverOpen(false)}
+    >
+      <EuiText size="s" grow={false}>
+        <p>
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.whenHelpText"
+            defaultMessage="The enrollment token is automatically generated when you start Elasticsearch for the first
+          time."
+          />
+        </p>
+        <p>
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.whereHelpText"
+            defaultMessage="You might need to scroll back a bit in the terminal to view the enrollment token."
+          />
+        </p>
+        <p>
+          <FormattedMessage
+            id="interactiveSetup.enrollmentTokenHelpPopover.howHelpText"
+            defaultMessage="To generate a new enrollment token run the following command in the Elasticsearch
+          directory:"
+          />
+        </p>
+        <EuiCodeBlock language="bash" paddingSize="m" isCopyable>
+          bin/elasticsearch-create-enrollment-token --scope kibana
```

Comment:
```suggestion
          bin/elasticsearch-create-enrollment-token -s kibana
```

Reply:
I think that we should use the shortened parameter option to align with the `elasticsearch-reset-password` command, which uses `-u`. It's also less for the user to type, though they might just copy + paste the command.

Reply:
I prefer long parameters since they're self explanatory. `-s` is a bit cryptic and users would have to look up what it stands for. 

Totally agree that both code snippets should be either long or short-hand arguments.